demo/admin/src/common/MasterMenu.tsx

```diff
@@ -1,95 +1,206 @@
+                route: {
+                    path: "/project-snips/main-menu",
+                    component: MainMenu,
+                },
+            },
+        ],
+    },
+    {
+        primary: <FormattedMessage id="menu.system" defaultMessage="System" />,
+        icon: <Wrench />,
+        subMenu: [
+            {
+                primary: <FormattedMessage id="menu.publisher" defaultMessage="Publisher" />,
+                route: {
+                    path: "/system/publisher",
+                    component: PublisherPage,
+                },
+            },
+            {
+                primary: <FormattedMessage id="menu.cronJobs" defaultMessage="Cron Jobs" />,
+                route: {
+                    path: "/system/cron-jobs",
+                    component: CronJobsPage,
+                },
+            },
+            {
+                primary: <FormattedMessage id="menu.redirects" defaultMessage="Redirects" />,
+                route: {
+                    path: "/system/redirects",
+                    render: () => <RedirectsPage redirectPathAfterChange="/system/redirects" />,
+                },
+            },
+        ],
+    },
+    {
+        primary: <FormattedMessage id="menu.componentDemo" defaultMessage="Component demo" />,
+        icon: <Snips />,
+        route: {
+            path: "/component-demo",
+            component: ComponentDemo,
+        },
+    },
+    {
+        primary: <FormattedMessage id="menu.products" defaultMessage="Products" />,
+        icon: <Snips />,
+        subMenu: [
+            {
+                primary: <FormattedMessage id="menu.products" defaultMessage="Products" />,
+                route: {
+                    path: "/products",
+                    component: ProductsPage,
+                },
+            },
+            {
+                primary: <FormattedMessage id="menu.productCategories" defaultMessage="Categories" />,
+                route: {
+                    path: "/product-categories",
+                    component: ProductCategoriesPage,
+                },
+            },
+            {
+                primary: <FormattedMessage id="menu.productTags" defaultMessage="Tags" />,
+                route: {
+                    path: "/product-tags",
+                    component: ProductTagsPage,
+                },
+            },
+            {
+                primary: <FormattedMessage id="menu.productsHandmade" defaultMessage="Products Handmade" />,
+                route: {
+                    path: "/products-handmade",
+                    component: ProductsHandmadePage,
+                },
+            },
+        ],
+    },
+    {
+        primary: <FormattedMessage id="menu.userPermissions" defaultMessage="User Permissions" />,
+        icon: <Snips />,
+        route: {
+            path: "/user-permissions",
+            component: UserPermissionsPage,
+        },
+    },
+];
 
-export default MasterMenu;
+export default function () {
```

Comment:
Optional: Anonymous default exports are bad for HMR

Reply:
b239d47b492433d7bdaa7c674e676abff33309ae